sdk/go/common/resource/plugin/context.go

```diff
@@ -34,11 +34,12 @@ import (
 // associated OS resources can be cached, shared, and reclaimed as
 // appropriate. It also carries shared plugin configuration.
 type Context struct {
-	Diag       diag.Sink // the diagnostics sink to use for messages.
-	StatusDiag diag.Sink // the diagnostics sink to use for status messages.
-	Host       Host      // the host that can be used to fetch providers.
-	Pwd        string    // the working directory to spawn all plugins in.
-	Root       string    // the root directory of the context.
+	Diag       diag.Sink         // the diagnostics sink to use for messages.
+	StatusDiag diag.Sink         // the diagnostics sink to use for status messages.
+	Debugging  DebugEventEmitter // the debugging event emitter to use to signal the engine that the debugger started.
```

Comment:
Shouldn't this only be needed on the Host, not the Context?

Reply:
Yeah, I think it was on the Context just for convenience, but it's not needed there.  Changed that in my last commit.